app/jobs/hyku_addons/reindex_available_works_job.rb

```diff
@@ -21,7 +29,8 @@ def reindex(cname)
           next unless klass.count.positive?
 
           Rails.logger.debug "==== Queue ReindexModelJob for #{klass} ==="
-          HykuAddons::ReindexModelJob.perform_later(model_class, cname)
+          HykuAddons::ReindexModelJob.perform_later(model_class, cname, options: { cname_doi_mint: @cname_doi_mint }) if @cname_doi_mint.include?(cname)
```

Comment:
What do you think about just: 

                HykuAddons::ReindexModelJob.perform_later(model_class, cname, options: { cname_doi_mint: @cname_doi_mint })
                
This is because @cname_doi_mint always returns a fault of empty array

Reply:
I said something very similar @thenapking. but as the `options` hash is set to a blank hash by default, if could be a ternary. 

```options = @cname_doi_mint.include?(cname) ? { options: { cname_doi_mint: @cname_doi_mint } } : {}

HykuAddons::ReindexModelJob.perform_later(model_class, cname, options) ```

Reply:
Yep, this is clearer to read as well I think.